src/reducers/todos.js

```diff
@@ -0,0 +1,22 @@
+import { ADD_TODO, DELETE_TODO, EDIT_TODO } from '../constants/todo';
+import { OrderedMap, Record } from 'immutable'
```

Comment:
In this example I don't use `immutable`，maybe you can use Vanilla JavaScript instead `immutable`.